### ISSUE TITLE
"4 hidden features"

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#5219]
1. https://www.openstreetmap.org/edit#map=19/23.67430/120.79611
2. See "4 hidden features" at bottom
3. Click on it
4. Nothing happens

**Comment by jidanni** [comment]
It should open the Map Features side panel, where the user months ago disabled some features.

**Comment by JamesKingdom** [comment]
Highlighting over `hidden features` clearly shows what is hidden: 
![](https://i.glideservers.com/u/jam33ada80c-6cd6-411c-9017-c5e46fcbb596.png)

**Comment by jidanni** [comment]
Yes for me it said "4 landuse features" upon mouseover.
But that's it.
No way to figure out what to do about it.

It assumes that we have recently turned them off, and even remember that it was us that turned them off, and that it was not turned off by the layer by default etc. And know where to go to fix the problem.

**Comment by bhousel** [comment]
Yes, I agree - we're working on improving the layer settings and filtering soon, so when that's done this indicator should open the new filtering screen.

**Comment by matkoniecz** [comment]
This was changed and now clicking on this part of interface is doing the expected action

<img width="503" height="748" alt="Image" src="https://github.com/user-attachments/assets/4c717e44-1885-4e8c-9cd4-9a48bb13db5c" />
